src/main/java/com/minecolonies/coremod/entity/ai/basic/AbstractEntityAIInteract.java

```diff
@@ -291,7 +291,8 @@ public void searchForItems(final AxisAlignedBB boundingBox)
     {
         items = world.getEntitiesWithinAABB(EntityItem.class, boundingBox)
                   .stream()
-                  .filter(item -> item != null && !item.isDead)
+                  .filter(item -> item != null && !item.isDead &&
+                          (!item.getEntityData().hasKey("PreventRemoteMovement") || !item.getEntityData().getBoolean("PreventRemoteMovement")))
```

Comment:
Shouldn't `!item.getEntityData().getBoolean("PreventRemoteMovement")` be `item.getEntityData().getBoolean("PreventRemoteMovement")`?

Reply:
or the `||` be an end `&&`?

Reply:
Great that this is settled =)